providers/keap/connector.go

```diff
@@ -0,0 +1,74 @@
+package keap
+
+import (
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/interpreter"
+	"github.com/amp-labs/connectors/common/paramsbuilder"
+	"github.com/amp-labs/connectors/common/urlbuilder"
+	"github.com/amp-labs/connectors/providers"
+	"github.com/amp-labs/connectors/providers/keap/metadata"
+)
+
+const ApiPathPrefix = "crm/rest"
```

Comment:
Is this the same for all modules and the entire API? Are there other products that have differing API prefixes?

Reply:
So while working on metadata PR, I dicsovered that I covered one of the two of OpenAPI files. First is V1 and the other much smaller is V2. I revisited the read from this persective and added some changes, in particular 2 modules.

To answer above question, yes, both share `crm/rest`.
Ex:
`crm/rest/v1` - first module
`crm/rest/v2` - second module